skipruntime-ts/ts/src/skipruntime_service.ts

```diff
@@ -51,13 +52,14 @@ export type OutputDefinition = {
 };
 
 export interface GenericSkipService {
-  // name / duration in milliseconds
-  tokens?: Record<string, number>;
   localInputs(): Record<string, InputDefinition>;
   remoteInputs(): Record<string, RemoteInputs>;
 
   outputs(): Record<string, OutputDefinition>;
 
+  // name / duration in milliseconds
```

Comment:
Both name and duration are present, right, not one or the other? If so:
```suggestion
  // name, duration in milliseconds
```